modules/ROOT/pages/runtime-configure-eu.adoc

```diff
@@ -22,20 +22,21 @@ wrapper.java.additional.<n>=-Danypoint.platform.base_uri=https://eu1.anypoint.mu
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
 ----
 
-In the above example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
+In the previous example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
 
 === Command Line
 
-To configure Mule Runtime from the command line, use the following options when starting the Mule runtime:
+To configure Mule runtime engine from the command line, use the following options when starting Mule runtime engine. Note that `./` only applies to Unix environments.
```

Comment:
Is it nec. to specify "...when starting the Mule runtime engine"? Does it change the meaning of the sentence to omit it (to configure, when starting)? It sounds a little odd, but that might be just the repetition of the full spelling.

Reply:
Alternative:
```suggestion
To configure Mule runtime engine from the command line, use the following options (note that `./` applies only to Unix environments):
```

Reply:
Leaving as is.